Formula/tinyscheme.rb

```diff
@@ -20,35 +22,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -?", 1)
```

Comment:
We're not certain if it's a broken binary, since @ilovezfs couldn't reproduce it locally. The failure message also didn't make any sense (even for a broken binary), so there's at least a good chance that the fault lies somewhere in CI rather than `tinyscheme`. Until resolved either way, it's probably a good idea to hold off on merging this one.